.ci_support/win_64_.yaml

```diff
@@ -0,0 +1,28 @@
+bzip2:
+- '1'
+c_compiler:
+- vs2019
+c_stdlib:
+- vs
+channel_sources:
+- conda-forge
+channel_targets:
+- conda-forge main
+cxx_compiler:
+- vs2019
+fortran_compiler:
+- flang
+fortran_compiler_version:
+- '5'
```

Comment:
Note that you're still adding fortran compilers here (seems to be based on an old rerender before adding the selector to the fortran compiler). However, since the dependency is missing in `meta.yaml`, this isn't actually pulled into the build environment for now.

Reply:
I would think we should remove this now.